src/main/java/com/minecolonies/coremod/commands/GetColonyAndCitizen.java

```diff
@@ -0,0 +1,157 @@
+package com.minecolonies.coremod.commands;
+
+import com.minecolonies.coremod.colony.ColonyManager;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+/**
+ * Get the Colony ID and Citizen ID out of a command.
+ */
+
+public final class GetColonyAndCitizen
+{
+
+    private static final String ONLY_NUMBERS                = "Please only use numbers for the %s ID!";
+    private static final String TOO_MANY_ARGUMENTS          = "Too many arguments!";
+    private static final String UNKNOWN_ERROR               = "Unknown Error!";
+    private static final String NOT_FOUND                   = "%s not found!";
+    private static final String NO_COLONY                   = "You haven't got a colony!";
+    private static final int    SHORT_ARGUMENT_LENGTH       = 1;
+    private static final int NORMAL_ARGUMENT_LENGTH      = 2;
+    private static final int NAME_ARGUMENT_LENGTH        = 3;
+    private static final int ID_AND_NAME_ARGUMENT_LENGTH = 4;
+    private static final int TOO_MANY_ARGUMENTS_LENGTH   = 5;
+    private static final int ARGUMENT_ZERO               = 0;
+    private static final int ARGUMENT_ONE                = 1;
+    private static final int ARGUMENT_TWO                = 2;
+    private static final int ARGUMENT_THREE              = 3;
+    private static final int STANDARD_CITIZEN_ID         = 0;
+
+    private GetColonyAndCitizen()
+    {
+        throw new IllegalAccessError("Utility Class");
+    }
+
+    /**
+     * Getting the colony ID.
+     * @param mayorID   The ID of the mayor.
+     * @param world     The world.
+     * @param args      The arguments.
+     * @return Return colony ID.
+     */
+    public static int getColonyId(@NotNull final UUID mayorID, @NotNull final World world, @NotNull final String... args)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getColonyId" is 15 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)